WordPress/openverse
# Initialize the stores only on the server

## Fixes



Fixes #5037 by @obulat

## Description


This PR makes the `init-stores` plugin run only on the server.

We don't need to run it both on the server and on the client because the cookie values don't change from the server to client rendering. We also do not need to re-fetch the providers since they were fetched on the server.

This PR also adds numbers to the plugin names to ensure that the order of execution is correct on the server: `01.api-token` - `02.init-stores` - other plugins. The token is necessary for fetching the providers, and the stores need to be initialized before anything else

The other plugins do not depend on each other, so there I removed numbers from their names.

## Testing Instructions

Run the app and check the browser devtools Network tab for requests. There should be no `stats` requests since they are sent on the server (and thus, don't show up in the browser devtools).

## Checklist


      PRs) or the media properties generator (`ov just catalog/generate-docs media-props`
      for the catalog or `ov just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
